File: Examen-Servidor/Examen-Servido/api/models/SistemaOperativo.js
/**
 * SistemaOperativo.js
 *
 * @description :: A model definition.  Represents a database table/collection/etc.
 * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
 */

module.exports = {

  attributes: {
    nombreSo:{type: "string"},
    versionApi:{type: "number"},
    fechaLanzamiento:{type: "string"},
    pesoGigasSo:{type: "number"},
    instalado:{type: "number"},

    aplicaciones:{
      collection:"Aplicaciones",
      via:"sistemaOperativoId"
    },

    usuario:{
      model: 'Usuario',

    }
  },

};

File: app/src/main/java/com/example/marcelo/mn_moviles_examen_1b/SistemaOperativo.kt
package com.example.marcelo.mn_moviles_examen_1b

import java.util.*

class SistemaOperativo(var nombreSo: String,
                       var versionApi: Int,
                       var fechaLanzamiento: String,
                       var pesoGigasSo: Double,
                       var instalado: Int,
                       var createdAt: Long,
                       var updatedAt: Long,
                       var id: Int,
                       var aplicaciones: List<Aplicaciones>) {
    var createdAtDate = Date(updatedAt)
    var updatedAtDate = Date(createdAt)


}